src/sentry/tasks/deletion.py

```diff
@@ -15,7 +15,6 @@
 # in prod we run with infinite retries to recover from errors
 # in debug/development, we assume these tasks generally shouldn't fail
 MAX_RETRIES = 1 if settings.DEBUG else None
```

Comment:
```suggestion
MAX_RETRIES = 1 if settings.DEBUG else 5
```

Reply:
See https://github.com/getsentry/sentry/issues/19721#issuecomment-716528370 for context.